bundles/org.openhab.binding.venstarthermostat/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -89,6 +146,141 @@
 		<state readOnly="true"/>
 	</channel-type>
 
+	<channel-type id="fanMode">
+		<item-type>String</item-type>
+		<label>Fan Mode</label>
+		<description>Current Fan Mode</description>
+		<state readOnly="false">
+			<options>
+				<option value="auto">Auto</option>
+				<option value="on">On</option>
+			</options>
+		</state>
+	</channel-type>
+
+	<channel-type id="fanModeRaw" advanced="true">
+		<item-type>Number</item-type>
+		<label>Fan Mode (Raw)</label>
+		<description>Current Fan Mode, as an integer number</description>
+		<state readOnly="true"/>
+	</channel-type>
+
+	<channel-type id="fanState">
+		<item-type>String</item-type>
+		<label>Fan State</label>
+		<description>Current Fan State</description>
+		<state readOnly="true">
+			<options>
+				<option value="off">Off</option>
+				<option value="on">On</option>
+			</options>
+		</state>
+	</channel-type>
```

Comment:
Why not rather use a switch type ?
Just a comment, not a request for change.

Reply:
No idea. I'm learning here, and I know practically nothing about xml. What would that look like?

Reply:
@digitaldan : is it ok for you like that ? I believe it should rather be a switch.

Reply:
I would propose something like:
```
	<channel-type id="fanState">
		<item-type>Switch</item-type>
		<label>Fan State</label>
		<description>Current Fan State</description>
		<state readOnly="true" />
	</channel-type>
```
But of course, it would also require few little changes in the thing handler.

Reply:
Ah sorry I misunderstood your comment. I'm sure there was a reason I chose a string. Let me think.

Reply:
Fan state does match a switch type quite clearly as its state is ON/OFF so yes i would agree with using a switch.

Reply:
@raveydavies : is it clear for you ? You will certainly have to update your handler (handle command and update state) but the change is easy.

Reply:
@lolodomo Yes it's clear, working on it now, sorry didn't have much time the last few days